README.md

```diff
@@ -7,9 +7,26 @@ Labyrinth
 [![Percentage of issues still open](http://isitmaintained.com/badge/open/fossasia/labyrinth.svg)](http://isitmaintained.com/project/fossasia/labyrinth "Percentage of issues still open")
 [![license](https://img.shields.io/github/license/fossasia/labyrinth.svg)](LICENSE)
 
+[![GitHub forks](https://img.shields.io/github/forks/fossasia/labyrinth.svg?style=social&label=Fork)]()
+[![GitHub stars](https://img.shields.io/github/stars/fossasia/labyrinth.svg?style=social&label=Stars)]()
+[![GitHub watchers](https://img.shields.io/github/watchers/fossasia/labyrinth.svg?style=social&label=Watch)]()
+
+
 [**Play Now**](http://rawgit.com/fossasia/labyrinth/master/index.html) | 
 [**Learn How to Play**](http://rawgit.com/fossasia/labyrinth/master/howtoplay.html)
 
+## Content Outline
+- [**Motivation**](https://github.com/fossasia/labyrinth#motivation)
+- [**Implementation**](https://github.com/fossasia/labyrinth#implementation)
```

Comment:
I like to do the following:
1. use relative links.
2. Motivation would look like this:

```
- [**Motivation**][motivation]

Reply:
And there you add: https://github.com/fossasia/labyrinth/pull/305/files#diff-04c6e90faac2675aa89e2176d2eec7d8L21

```
 Motivation
 ----------
 [motivation]: #motivation
```

This way: if we change the heading, we can change the link right below it and all is together.